ddtrace/contrib/mako/__init__.py

```diff
@@ -0,0 +1,12 @@
+from ..util import require_modules
```

Comment:
We need to add a docstring here explaining the integration and how to use it.

Example: https://github.com/DataDog/dd-trace-py/blob/a0bdd4822127ce50e3776ca703e1266b0c36ab6d/ddtrace/contrib/flask/__init__.py#L1-L89